packages/cli/lib/utils.js

```diff
@@ -103,8 +103,13 @@ exports.validateClassName = function(name) {
 
 /**
  * Validate project directory to not exist
+ * and check if the directory path is based on CWD
  */
 exports.validateNotExisting = function(projDir) {
+  const invalidPath = projDir.split('/').includes('~');
+  if (invalidPath) {
+    return util.format('The path shold be based on CWD: %s. ', projDir);
```

Comment:
nitpick: since users might not know what CWD is I suggest `current directory`

Reply:
yes I agree. I wasn't sure about the warning message. And I will add tests later.